### ISSUE TITLE
Buy from dapp

### ISSUE THREAD

**Issue by Ponda1** [HashLips/frontend_base_dapp#1]
How can others use the dapp to buy my nfts?

**Comment by Jharmony** [comment]
You can follow HashLip's tutorial here:  https://www.youtube.com/watch?v=SD1DTrlJeKM Where you can follow along as he explains how to store your .png and .json file and then how you an create the contract to allow minting of your NFTs ( buying he nft) through you dapp.

**Comment by Auxetics** [comment]
> You can follow HashLip's tutorial here: https://www.youtube.com/watch?v=SD1DTrlJeKM Where you can follow along as he explains how to store your .png and .json file and then how you an create the contract to allow minting of your NFTs ( buying he nft) through you dapp.

Yeah I'm at a loss even after watching the final video in the 4 part series on what to modify in this to get it up and running on my website since the code in the video is slightly different than whats available for download. I'm also looking to use it on ETH and not polygon so its unclear if i need to modify anything.. that and i'm getting an error running yarn build.
